providers/github/resources/discovery.go

```diff
@@ -102,7 +102,6 @@ func org(runtime *plugin.Runtime, orgName string, conn *connection.GithubConnect
 		Connections: []*inventory.Config{conf.Clone(inventory.WithoutDiscovery(), inventory.WithParentConnectionId(conn.ID()))},
 	})
 	if stringx.ContainsAnyOf(targets, connection.DiscoveryRepos, connection.DiscoveryRepository, connection.DiscoveryAll, connection.DiscoveryAuto) {
-		assetList = []*inventory.Asset{}
```

Comment:
this resets the slice with discovered assets from above, meaning we miss the org itself